lib/DDG/Spice/BrainyQuote.pm

```diff
@@ -9,10 +9,16 @@ triggers startend => 'quote', 'quotes', 'quotation', 'quotations';
 spice to => 'https://www.brainyquote.com/api/ddg?q=$1';
 spice wrap_jsonp_callback => 1;
 
+spice alt_to => {
```

Comment:
Ah, my mistake -- The `alt_to` is necessary to force a JSON response, instead of JSONP.

The name should be something better than `test_endpoint` though 👍